doc/whats_new/v1.0.rst

```diff
@@ -646,6 +646,15 @@ Changelog
 - |Enhancement| warn only once in the main process for per-split fit failures
   in cross-validation. :pr:`20619` by :user:`Loïc Estève <lesteve>`
 
+
+:mod:`sklearn.multiclass`
+.........................
+
+- |Fix| The `n_features_in_` attribute of :class:`multiclass.OneVsOneClassifier`
+  was incorrect when fit with a base estimator that expects a precomputed
+  kernel matrix.
+  :pr:`20853` by :user:`Olivier Grisel <ogrisel>`.
+
```

Comment:
```suggestion
```
This was introduced in #20193 and was not yet released. So we don't need a whatsnew entry.

Reply:
Good catch.